_test/helper.test.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+class helper_plugin_captcha_public extends helper_plugin_captcha {
+
+    public function get_field_in() {
+        return $this->field_in;
+    }
+    public function get_field_sec() {
+        return $this->field_sec;
+    }
+    public function get_field_hp() {
+        return $this->field_hp;
+    }
+}
+
+/**
+ * @group plugin_captcha
+ * @group plugins
+ */
+class helper_plugin_captcha_test extends DokuWikiTest {
+
+    protected $pluginsEnabled = array('captcha');
+
+    public function testConfig() {
+        global $conf;
+        $conf['plugin']['captcha']['lettercount'] = 20;
+
+        $helper = new helper_plugin_captcha_public();
+
+        $code = $helper->_generateCAPTCHA("fixed", 0);
+        $this->assertEquals(16, strlen($code));
```

Comment:
Is returning a short string even the right thing to do? The extra digits will always be 0 so more characters are meaningless.I was thinking of a case where the displayed CAPTCHA is different than what was generated. But now I don't know if that's possible.